spec/supplejack/story_spec.rb

```diff
@@ -355,6 +355,20 @@ module Supplejack
 
         expect(Supplejack::Story.all_public_stories.count).to eq(2)
       end
+
+      it 'includes stories metadata if :meta_included option passed' do
+        expect(Supplejack::Story).to receive(:get).and_return(
```

Comment:
Would you be able to stub the request a step up to return the raw JSON and then see that it gets altered how you are expecting?